lib/modules/hosts/flickr.js

```diff
@@ -1,8 +1,12 @@
 addLibrary('mediaHosts', 'flickr', {
-	domains: ['flickr.com'],
+	domains: ['flickr.com', 'flic.kr'],
 	logo: '//s.yimg.com/pw/favicon.ico',
-	detect: href => (/^https?:\/\/(?:\w+\.)?flickr\.com\/(?:.+)\/(\d{10,})(?:\/|$)/i).test(href),
+	detect: href => ((/^https?:\/\/(?:\w+\.)?flickr\.com\/(?:.+)\/(\d{10,})(?:\/|$)/i).test(href) || (/^https?:\/\/(?:\w+\.)?flic\.kr\/p\/(\w+)(?:\/|$)/i).test(href)),
 	async handleLink(elem) {
+		// noembed.com does not yet support flickr shortlink.
+		// This fix can be removed once it's fixed.
+		elem.href = elem.href.replace('flic.kr', 'flickr.com');
```

Comment:
We don't want to assign to `elem.href` cause it's the actual link element on the page.
I'd be fine with either `.replace`ing inline in `data: { url: elem.href },` or assigning this to `const url`, whatever you want.